src/app/public/modules/ag-grid/cell-editors/cell-editor-autocomplete/cell-editor-autocomplete.component.spec.ts

```diff
@@ -0,0 +1,198 @@
+import {
+  async,
+  ComponentFixture,
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  SkyAppTestModule
+} from '@skyux-sdk/builder/runtime/testing/browser';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  Column,
+  ICellEditorParams,
+  RowNode
+} from 'ag-grid-community';
+
+import {
+  SkyAgGridCellEditorAutocompleteComponent
+} from '../cell-editor-autocomplete';
+
+describe('SkyCellEditorAutocompleteComponent', () => {
+  let fixture: ComponentFixture<SkyAgGridCellEditorAutocompleteComponent>;
+  let component: SkyAgGridCellEditorAutocompleteComponent;
+  let nativeElement: HTMLElement;
+
+  const data = [
+    { id: '1', name: 'John Doe', town: 'Daniel Island' },
+    { id: '2', name: 'Jane Doe', town: 'Daniel Island' },
+    { id: '3', name: 'John Smith', town: 'West Ashley' },
+    { id: '4', name: 'Jane Smith', town: 'Mt Pleasant' }
+  ];
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        SkyAppTestModule
+      ]
+    });
+
+    fixture = TestBed.createComponent(SkyAgGridCellEditorAutocompleteComponent);
+    nativeElement = fixture.nativeElement;
+    component = fixture.componentInstance;
+  });
+
+  it('renders an autocomplete input', () => {
```

Comment:
The test description should be "should render an autocomplete input" for consistency.